wrappers/javascript/anoncreds-nodejs/README.md

```diff
@@ -0,0 +1,26 @@
+# anoncreds-nodejs
```

Comment:
Can you make this consistent with askar / vdr?

Reply:
Done.